presto-tests/src/test/java/com/facebook/presto/execution/TestWarnings.java

```diff
@@ -150,6 +150,19 @@ public void testMultipleOrderByWarnings()
                 queryRunner, TEST_SESSION, query, ImmutableSet.of(MULTIPLE_ORDER_BY.toWarningCode()));
     }
 
+    @Test
+    public void testLikeOrCombinationWarnings()
+    {
+        String query = "SELECT * FROM customer WHERE name LIKE '%abcd%' OR name LIKE '%efgh%'";
```

Comment:
Can we also add test cases with multiple `OR`, mixed `AND` and `OR, and where complex expression is used? Thanks!